src/EditorFeatures/Test2/InlineHints/CSharpInlineTypeHintsTests.vb

```diff
@@ -766,7 +766,7 @@ class A
                     <Document>
 class A
 {
-    void M(System.Threading.CancellationToken ct = new CancellationToken()) { }
+    void M(System.Threading.CancellationToken ct = new System.Threading.CancellationToken()) { }
```

Comment:
this is a desirable change.  The code emit before was emitting code that wouldn't compile.